src/harness/harness.c

```diff
@@ -36,49 +36,884 @@ tHarness_game_config harness_game_config;
 // Platform hooks
 tHarness_platform gHarness_platform;
 
-/* clang-format off */
+// Carmgeddon ASCII codes
```

Comment:
how about putting these tables in a separate file like `ascii_tables.h` to avoid adding all this data to the code file?

Reply:
I did something similar with `sdl2_scancode_to_dinput.h`

Reply:
I created a `ascii_tables.h` header.